main.js

```diff
@@ -0,0 +1,15 @@
+const menu = document.querySelector('#union');
+const navigation = document.querySelector('nav');
+menu.addEventListener('click', () => {
+  menu.classList.toggle('fa-times');
```

Comment:
- [ ]  You have done an excelent job, but please make sure that on desktop size the hamburger menu button is not visible. 📈✏️

![image](https://user-images.githubusercontent.com/95243996/201699835-ecefe784-0b01-4fb5-83e2-4688b5570606.png)

Reply:
Thank you so much for the review. I have [resolved ](https://github.com/Lucash2022/LucasErkanaPortfolio_Website/pull/16/commits/0efb22eefd31fd5f7ae21c6602b971febd2c5e62)the issue as [requested.](https://github.com/Lucash2022/LucasErkanaPortfolio_Website/pull/16/commits/0efb22eefd31fd5f7ae21c6602b971febd2c5e62)